fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/ambassador/AbstractSumoAmbassador.java

```diff
@@ -1283,6 +1284,18 @@ private void removeExternalVehiclesFromUpdates(VehicleUpdates updates) {
         updates.getRemovedNames().removeIf(vehicle -> externalVehicles.remove(vehicle) != null);
     }
 
+    public void changeExternalParameters(VehicleParametersChange vehicleParametersChange) throws InternalFederateException {
+        final String veh_id = vehicleParametersChange.getVehicleId();
+        for (final VehicleParameter param : vehicleParametersChange.getVehicleParameters()) {
+            if (param.getParameterType() == VehicleParameter.VehicleParameterType.COLOR) {
+                final Color color = param.getValue();
+                bridge.getVehicleControl().setColor(veh_id, color.getRed(), color.getGreen(), color.getBlue(), color.getAlpha());
+            } else {
+                log.warn("Parameter type {} is not supported for external vehicles", param.getParameterType().name());
```

Comment:
Ok, given your explanation from the previous comment we can keep that here. However, we previously fully ignored the VehicleParametersChange for external vehicles but now we print a warning for every unsupported parameter. I'd rather propose to omit the log here completely (or collect all unsupported paramterType-names and print them in a `debug` log at the end of the method, but imho not putting any log here would be fine too)